scripts/SANS/SANSadd2.py

```
@@ -144,55 +142,51 @@ def add_runs(  # noqa: C901
-                if isFirstDataSetEvent:
+                if is_first_dataset_event:
                     DeleteWorkspace(ADD_FILES_NEW_TEMPORARY_MONITORS)
                 # Increment the run number
                 counter_run += 1
         except ValueError as e:
-            error = "Error opening file {}: {}".format(userEntry, str(e))
-            print(error)
-            sanslog.error(error)
-            if ADD_FILES_SUM_TEMPORARY in mtd:
-                DeleteWorkspace(ADD_FILES_SUM_TEMPORARY)
+            _report_error(f"Error opening file {user_entry}:{str(e)}")
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)